src/components/ConnectToXeroFlow/index.native.tsx

```diff
@@ -40,17 +40,15 @@ function ConnectToXeroFlow({policyID}: ConnectToXeroFlowProps) {
 
     return (
         <>
-            {isRequire2FAModalOpen && (
```

Comment:
Why can't we apply same change to web based platforms?

Reply:
Yeah, you're right! It's not so visible on web but still is a little 🔍